test/common/filesystem/filesystem_impl_test.cc

```diff
@@ -121,6 +121,43 @@ TEST_F(FileSystemImplTest, CanonicalPathFail) {
 }
 #endif
 
+TEST_F(FileSystemImplTest, splitPathFromFilename) {
```

Comment:
nit: `SplitPathFromFilename`

Reply:
Turned out to be a bit more of an ordeal but that file's test names are now addressed.